src/uct/ib/rc/base/rc_iface.c

```diff
@@ -14,7 +14,7 @@
 
 
 ucs_config_field_t uct_rc_iface_config_table[] = {
-  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=4095", NULL,
+  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=256", NULL,
```

Comment:
The task is to reduce overall memory consumption by default for all possible applications. it should influence on startup time and scalability.